solutions/bronze/usaco-1227.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: usaco-1227
+source: USACO Bronze 2022 US Open
+title: Photoshoot
+author: Kevin Sheng
+---
+
+<Spoiler title="Hint 1">
+
+Not caring about the number of flips, what's the best FJ can do in terms
+of putting cows in their even/odd positions?
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+If we reverse an odd number of cows, does anything effectively change?
+
+</Spoiler>
+
+<Spoiler title="Hint 3">
+
+How can we process the cows such that we don't have to look back to already
+processed cows? Keep in mind that we can only reverse *prefixes*.
+
+</Spoiler>
+
+<Spoiler title="Solution">
+
+[Official Analysis (C++, Java, Python)](http://usaco.org/current/data/sol_prob1_bronze_open22.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <cassert>
+#include <string>
+
+using std::cout;
+using std::endl;
+using std::string;
+
+int main() {
+	int cow_num;
+	string cows;
+	std::cin >> cow_num >> cows;
+	assert(cows.size() == cow_num && cow_num % 2 == 0);
```

Comment:
whats the point of this

Reply:
i let you keep your `n`, you let me keep this